app/views/organisations/index.html.erb

```diff
@@ -6,17 +6,13 @@
       <tr>
         <th>Name</th>
         <th>Address</th>
-        <th>Email</th>
-        <th>Contact No.</th>
```

Comment:
what is this change about?

Reply:
the page was set as public so i removed the emails from it.